charts/gsp-cluster/templates/02-gsp-system/service-operator/service.yaml

```diff
@@ -0,0 +1,20 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Release.Name }}-service-operator
+  labels:
+    app.kubernetes.io/name: {{ .Release.Name }}-service-operator
+    helm.sh/chart: {{ .Chart.Name }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    control-plane: service-operator
+    controller-tools.k8s.io: "1.0"
+spec:
+  ports:
```

Comment:
This doesn't look istio-compliant
https://istio.io/docs/setup/kubernetes/additional-setup/requirements/
I was also surprised to see an https listener on this operator...
Is a `Service` needed at all for this?